bom/runtime/pom.xml

```diff
@@ -2315,25 +2315,6 @@
                 <artifactId>aws-lambda-java-events</artifactId>
                 <version>${aws-lambda-java-events.version}</version>
             </dependency>
-            <dependency>
-                <groupId>com.amazonaws.serverless</groupId>
-                <artifactId>aws-serverless-java-container-core</artifactId>
-                <version>${aws-lambda-serverless-java-container.version}</version>
```

Comment:
Is the `aws-lambda-serverless-java-container.version` property still used?

Reply:
no, it isn't.  I'll remove it.